client/src/components/Modals/SubmitPitch/index.tsx

```diff
@@ -131,109 +145,136 @@ const SubmitPitchModal: FC<SubmitPitchModalProps> = ({
+                    label="Yes"
+                    onClick={() => setWriterChoice(true)}
+                    checked={writerChoice}
+                    error={isFieldError(writerChoice)}
+                  />
+                  <Form.Radio
+                    name="writer-choice"
+                    label="No, I want someone else to write the story"
+                    onClick={() => setWriterChoice(false)}
+                    checked={!isUndefined(writerChoice) && !writerChoice}
+                    error={isFieldError(writerChoice)}
+                  />
+                </Form.Group>{' '}
+              </div>
+            ) : (
+              <></>
+            )}
 
-          <h3>Topics (Please select at least 1 topic)</h3>
-          <Form.Group inline widths="4" className="checkbox-group">
-            {interests.map((topic, index) => (
-              <Form.Checkbox
-                error={isFieldError(topics)}
-                className="checkbox"
-                key={index}
-                label={titleCase(topic.name)}
-                checked={topics.has(topic._id)}
-                onClick={() => addTopic(topic._id)}
-                value={topic._id}
+            <h5 className="label">Conflict of Interest Disclosure</h5>
+            <p>
+              Are you involved with the events or people covered in your pitch?
+              i.e. do you have a relationship with them as an employee, family
+              member, friend, or donor?
+            </p>
+            <Form.Group inline>
+              <Form.Radio
+                name="interest-conflict"
+                label="Yes"
+                onClick={() => setConflictOfInterest(true)}
+                checked={conflictofInterest}
+                error={isFieldError(conflictofInterest)}
               />
-            ))}
-          </Form.Group>
-          <h3>Conflict of Interest Discolsure</h3>
-          <p>
-            Are you involved with the events or people covered in your pitch?
-            i.e. do you have a relationship with them as an employee, family
-            member, friend, or donor?
-          </p>
-          <Form.Group inline>
-            <Form.Radio
-              name="interest-conflict"
-              label="Yes"
-              onClick={() => setConflictOfInterest(true)}
-              checked={conflictofInterest}
-              error={isFieldError(conflictofInterest)}
-            />
-            <Form.Radio
-              name="interest-conflict"
-              label="No"
-              onClick={() => setConflictOfInterest(false)}
-              checked={!isUndefined(conflictofInterest) && !conflictofInterest}
-              error={isFieldError(conflictofInterest)}
+              <Form.Radio
+                name="interest-conflict"
+                label="No"
+                onClick={() => setConflictOfInterest(false)}
+                checked={
+                  !isUndefined(conflictofInterest) && !conflictofInterest
```

Comment:
I don't know if you are the one who added this, I don't think it was tbh - it was probably me. But I think that this can just be `checked={!conflictofInterest}` because undefined returns false

Reply:
I think we have to keep this so it doesn't default check false in the beginning